gridstatus/pjm.py

```diff
@@ -2988,10 +2988,10 @@ def get_projected_peak_tie_flow(
             },
             interval_duration_min=60,
             verbose=verbose,
-            filter_timestamp_name="projected_peak_datetime",
+            filter_timestamp_name="generated_at_ept",
```

Comment:
We shouldn't need the `_ept` here since it's added in `_get_pjm_json`.

https://github.com/gridstatus/gridstatus/blob/6645f55516f88fa69dc029435ad9ad4370c5a2b9/gridstatus/pjm.py#L915-L917